merge-request-status.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Creates a new column for merge request status.
+ *
+ * @type {{createElement: (function(): HTMLElement)}}
+ */
+import { utils } from "./common.js";
+
+const mergeRequestStatus = {
+  createElement: function () {
+    // store name elements in array-like object
+    const namesFromDOM = document.querySelectorAll(
+      "tbody .views-field-title a"
+    );
+    for (const nameElement of namesFromDOM) {
+      const issueId = utils.getIssueIdFromUrl(nameElement.getAttribute("href"));
+
+      const address = fetch(
+        `https://git.drupalcode.org/api/v4/merge_requests?state=opened&scope=all&in=title&search=` +
+          issueId
+      )
+        .then((response) => response.json())
+        .then((data) => {
+          const tdElement = document.createElement("td");
+          if (Object.keys(data).length === 0) {
+            tdElement.appendChild(document.createTextNode("No"));
+          } else {
+            for (const key in data) {
+              const anchorLink = document.createElement("a");
+              const link = document.createTextNode(
+                "#" + data[key].iid + " " + data[key].merge_status + " Total comment count " + data[key].user_notes_count
```

Comment:
what do you think about this?
```suggestion
               "#" + data[key].iid + " " + data[key].merge_status + `, ${data[key].user_notes_count} comments`
```

just because it is a lot of text in the column